docs/src/pages/index.page.tsx

```diff
@@ -201,22 +201,22 @@ const HomePage = ({ colorMode, setThemeOverride, themeOverride }) => {
             display={{ base: 'none', large: 'initial' }}
           >
             <Image
-              alt=""
-              className="docs-home-vue"
-              src="/svg/integrations/vue.svg"
-            />
-            <Image
-              alt=""
+              alt="React framework image"
```

Comment:
Alt text shouldn't have the word image in it, since the screen reader already knows it's an image. Alt text is meant to replace the image if it doesn't show up, so maybe `React logo` would make more sense here.